### ISSUE TITLE
How do I get this file?

### ISSUE THREAD

**Issue by Aran-Guo** [zhuchenxv/AutoFIS#3]
I don't understand where this file came from.Do I need to make the switch myself?

**Comment by Aran-Guo** [comment]
This is the error : 'Unable to open/create file 'D:\论文复现\AutoFIS\Avazu\hdf\train_output_part_0.h5'

**Comment by zhuchenxv** [comment]
The dataset file could be found from https://github.com/Atomu2014/Ads-RecSys-Datasets. It also tells you how to use. Notes are in AutoFIS/dataset.txt.

**Comment by sunupo** [comment]
> The dataset file could be found from https://github.com/Atomu2014/Ads-RecSys-Datasets. It also tells you how to use. Notes are in AutoFIS/dataset.txt.

I have got this H5 fomat file of Avazu.
How can we get the data file (eg:raw_file='avazu.tr.svm',  input_feat_file='train_input.txt') rementioned in Avazu.py?
Is The file ( 'train_input.txt' )  similar to train.csv which get from the kaggle?

**Comment by zhuchenxv** [comment]
> > The dataset file could be found from https://github.com/Atomu2014/Ads-RecSys-Datasets. It also tells you how to use. Notes are in AutoFIS/dataset.txt.
> 
> I have got this H5 fomat file of Avazu.
> How can we get the data file (eg:raw_file='avazu.tr.svm', input_feat_file='train_input.txt') rementioned in Avazu.py?
> Is The file ( 'train_input.txt' ) similar to train.csv which get from the kaggle?

Yes, the raw dataset can be downloaded from https://www.kaggle.com/c/avazu-ctr-prediction/data. It is noted in the README.md in https://github.com/Atomu2014/Ads-RecSys-Datasets.

**Comment by sunupo** [comment]
> > > The dataset file could be found from https://github.com/Atomu2014/Ads-RecSys-Datasets. It also tells you how to use. Notes are in AutoFIS/dataset.txt.
> > 
> > 
> > I have got this H5 fomat file of Avazu.
> > How can we get the data file (eg:raw_file='avazu.tr.svm', input_feat_file='train_input.txt') rementioned in Avazu.py?
> > Is The file ( 'train_input.txt' ) similar to train.csv which get from the kaggle?
> 
> Yes, the raw dataset can be downloaded from https://www.kaggle.com/c/avazu-ctr-prediction/data. It is noted in the README.md in https://github.com/Atomu2014/Ads-RecSys-Datasets.

Thanks!
The raw dataset of avazu  with the label(" click ") has only single file called train.csv. Was its train/test sets split by you?
If so, what is the parameter to split this dataset?

**Comment by zhuchenxv** [comment]
> > > > The dataset file could be found from https://github.com/Atomu2014/Ads-RecSys-Datasets. It also tells you how to use. Notes are in AutoFIS/dataset.txt.
> > > 
> > > 
> > > I have got this H5 fomat file of Avazu.
> > > How can we get the data file (eg:raw_file='avazu.tr.svm', input_feat_file='train_input.txt') rementioned in Avazu.py?
> > > Is The file ( 'train_input.txt' ) similar to train.csv which get from the kaggle?
> > 
> > 
> > Yes, the raw dataset can be downloaded from https://www.kaggle.com/c/avazu-ctr-prediction/data. It is noted in the README.md in https://github.com/Atomu2014/Ads-RecSys-Datasets.
> 
> Thanks!
> The raw dataset of avazu with the label(" click ") has only single file called train.csv. Was its train/test sets split by you?
> If so, what is the parameter to split this dataset?

The raw dataset of avazu in https://www.kaggle.com/c/avazu-ctr-prediction/data has two files, train.csv and test.csv. We do not split the Avazu dataset and you could check again.

**Comment by takeawayls** [comment]
from datasets import as_dataset  
is missing,how i get ?

**Comment by zhuchenxv** [comment]
> from datasets import as_dataset
> is missing,how i get ?

The dataset file could be found from https://github.com/Atomu2014/Ads-RecSys-Datasets. It also tells you how to use. Notes are in AutoFIS/dataset.txt.
